fintech-examples/fintech-ui/proxy.conf.json

```diff
@@ -1,6 +1,6 @@
 {
   "/fintech-api-proxy/*": {
-    "target": "http://localhost:8080",
+    "target": "http://localhost:8086/v1/",
```

Comment:
I would add `/v1/` to your path in environment.ts.

Reply:
What's the benefit of that?